charts/kof-istio/templates/telemetry.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  namespace: istio-system
+  name: otel-tracing
+  annotations:
+    helm.sh/hook: post-install
+spec:
+  tracing:
+  - providers:
+    - name: otel-tracing
+    randomSamplingPercentage: 100
```

Let's add a value for this
```suggestion
    randomSamplingPercentage: {{ .Values.kof.traces_sampling_percentage }}
```